guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -21,25 +21,43 @@
 - Fine-tuning a pretrained backbone
 - Training a image classifier from scratch
 
+KerasCV uses Keras 3 to work with any of TensorFlow, Pytorch and Jax. In the
```

Comment:
"TensorFlow, Pytorch and Jax." -> "TensorFlow, PyTorch, or Jax." (Note spelling of "PyTorch".)

Reply:
Thanks. Done.